src/networkAddresses.ts

```diff
@@ -72,9 +72,18 @@ export const NETWORK_ADDRESSES = {
   opbnbtestnet: {
     NORMAL_TIMELOCK: "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf",
     GUARDIAN: "0xb15f6EfEbC276A3b9805df81b5FB3D50C2A62BDf",
+    XVS: "0xc2931B1fEa69b6D6dA65a50363A8D75d285e4da9",
+    XVS_VAULT_PROXY: "0xB14A0e72C5C202139F78963C9e89252c1ad16f01",
+    XVS_STORE: "0x06473fB3f7bF11e2E8EfEcC95aC55ABEFCb2e0A0",
+    GENERIC_TEST_USER_ACCOUNT: "0x2Ce1d0ffD7E869D9DF33e28552b12DdDed326706",
+    ACM: "0x049f77F7046266d27C3bC96376f53C17Ef09c986",
```

Comment:
I would remove the entries for `XVS_STORE` and `ACM`, because they are not used by the VIP framework. I would stop hardcoding addresses, and I would set them here using the addresses from the npm packages.